en/docs/choreo-samples/quick-deploy-a-sample.md

```diff
@@ -1,20 +1,20 @@
-# Quick Deploy a Sample
+# Quick deploy a sample
 
-Choreo provides a quick deploy option for samples, allowing you to deploy a sample with a single click. This capability lets you quickly explore the sample.
+Choreo provides a **Quick Deploy** option for samples, allowing you to deploy a sample with a single click. This feature enables you to quickly explore and experiment with the sample.
 
 ## Prerequisites
 
 Before you start using the samples, complete the following steps:
 
-- Sign in to Choreo and go to your project **Overview** page.
-   - If you're signing in for the first time, follow these steps to create an organization and then create a project:  
-      1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in using your preferred method.
-      2. Enter a unique organization name. For example, `Stark Industries`.
+1. **Sign in to Choreo** and go to your project **Overview** page.
+   - If you're signing in for the first time, follow these steps to create an organization and project:
+      1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in using your preferred method.
+      2. Enter a unique organization name (e.g., `Stark Industries`).
```

Comment:
To adhere to the Google style guide, we generally avoid using e.g., https://developers.google.com/style/word-list#eg